charts/actions-runner-controller/templates/deployment.yaml

```diff
@@ -122,6 +122,14 @@ spec:
         - name: {{ $key }}
           value: {{ $val | quote }}
         {{- end }}
```

Comment:
Now the above 4 lines are duplicates of L128-131 which results in invalid env when you've specified a slice as env:

```
$ helm template foo ./charts/actions-runner-controller --set env[0].name=env1,env[0].value=val1 | grep val1 -C 2
              optional: true
        - name: 0
          value: "map[name:env1 value:val1]"
        - name: env1
          value: val1
        image: "summerwind/actions-runner-controller:v0.24.1"
        name: manager
```

Reply:
fixed